setup.cfg

```diff
@@ -26,7 +26,8 @@ install_requires =
     nanomsg
     pyzmq
     selenium<4
-    uvloop
+    setuptools
+    uvloop==0.21.0b1
```

Comment:
hmm, do we know why we are pinning to a specific version, especially a beta one?  what are the conditions in the future under which we could remove the pin?